MIPS/mip-30.md

```diff
@@ -0,0 +1,385 @@
+
+The second principle is to remove all integration contracts such as Aave and Compound, meaning that all of the benefits associated with keeping funds within the contracts will be eliminated. This includes the ability to earn interest on deposits and any rewards. Furthermore, all fees will be set to zero to allow natural arbitrage to flow freely, without the user being able to earn, while also allowing for easier withdrawals with fewer fees in any basket asset.
+
+The third principle is to remove all ownership from contracts held by the ProtocolDAO. This means removing the ability to change parameters, upgrading contracts, and finally renouncing all ownership of the nexus contract and the delayed upgradable proxy.
+
+The fourth principle of open-source projects is to make any remaining repositories that have not yet been made publicly available to the community. This is an important step towards allowing other developers to benefit from the work of the project, as it allows them to access the code and build upon it.
+
+The fifth principle is to communicate the sunset of deployed apps, landing pages, and social channels. This will allow for withdrawals on the hosted front end for a period of at least 6 months starting after the vote of the sunset in the subsequent proposal has been conducted. After this, withdrawals can still be made using locally hosted front-ends or through direct smart contract interaction, such as Etherscan.
+
+## Motivation
+
+The motivation behind this MIP is to ensure a swift and secure transition for the sunset of the products while allowing users to remain in control of their funds and access them at any time in the future, removing ownership from contracts held by the ProtocolDAO on behalf of MTA Governors / mStable DAO.
+
+Data on Ethereum and Polygon don’t have an expiration date and our smart contracts should reflect this fact. Any user, no matter when they chose to withdraw, should get their assets back.
+
+It's important to keep in mind that any changes made to a system may require a significant amount of time to implement properly. Once the changes are in place, it's also essential to test them thoroughly to ensure that they function as intended. Only after thorough testing should the changes be deployed into the live environment to minimize the risk of any potential issues arising. All of these steps are critical to ensuring that changes are made safely and effectively.
+
+## Specification
+
+Outlined here is the high-level approach to the sunset of each product. Staking contracts are excluded here and will be further specified in another proposal outlining MTA procedure.
+
+### mAssets and Save
+
+- Upgrade the mAssets’s `migrateBassets` to support migrating bAssets from platforms back to the mAsset contract
+- Remove platform integration
+- Set swap and redeem fee to 0
+- Set new min-max weights (5% and 95%)
+- This applies to mUSD and mBTC on Ethereum mainnet and to mUSD on Polygon
```

Comment:
should we add something `mBTC(deprecated)`

Reply:
I don't think so, because we never actually resolved anything here, never changed the mBTC smart contracts.